api/desecapi/views.py

```diff
@@ -41,6 +46,52 @@ def get_client_ip(request):
     return request.META.get('REMOTE_ADDR')
 
 
+class TokenCreateView(djoser.views.TokenCreateView):
+
+    def _action(self, serializer):
```

Comment:
The original Djoser `login_user()` function in `utils.py` uses `settings.TOKEN_MODEL.objects.get_or_create()`. The right way to handle this would be to assign a non-standard `Manager` to the token model, so that `get_or_create()` does what you want to do (in the new `Token` model). Then, this view (which is essentially code duplication) is unnecessary.